docs/disconnected-updateservice-operator.md

```diff
@@ -0,0 +1,295 @@
+  --from-file=.dockerconfigjson=<path/to/.docker/config.json> \
+  --type=kubernetes.io/dockerconfigjson
+```
+
+## Deploy the UpdateService operator
+
+```
+oc create ns openshift-update-service
+cat <<EOF | oc create -f - 
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: cincinnati-subscription
+spec:
+  channel: v1
+  name: cincinnati-operator
+  installPlanApproval: Automatic
+  source: my-redhat-operator-catalog
+  sourceNamespace: openshift-marketplace
+---
+  apiVersion: operators.coreos.com/v1
+  kind: OperatorGroup
+  metadata:
+    name: cincinnati-operatorgroup
+  spec:
+    targetNamespaces:
+      - openshift-update-service
+EOF
+```
+
+## Create initial graph data container image 
+
+We need to create the graph data to allow upgrade between version. We can use the latest
+data or create our own. For details refer the [init container documentation](./graph-data-init-container.md).
+
+In order to build a container image that contains the graph-data, which will be used as init
+container in the updateservice pod: 
+
+```
+wget https://github.com/openshift/cincinnati-graph-data/archive/master.tar.gz -O graph.tar.gz
+cat <<EOF > Dockerfile
+FROM registry.access.redhat.com/ubi8/ubi:8.1
+COPY graph.tar.gz /tmp/
+RUN mkdir -p /var/lib/cincinnati/graph-data/
+CMD exec /bin/bash -c "tar xvzf /tmp/graph.tar.gz -C /var/lib/cincinnati/graph-data/ --strip-components=1"
+EOF
+
+podman build -f Dockerfile -t ${DISCONNECTED_REGISTRY}/updateservice/graph-data:v1
+podman push ${DISCONNECTED_REGISTRY}/updateservice/graph-data:v1
+```
+## Mirror the release image and the release content
+
+You might want to review the documentation around disconnected registries to learn about mirroring OpenShift Releases.
+
+```
+export UPSTREAM_REGISTRY='quay.io'
+export PRODUCT_REPO='openshift-release-dev'
+export RELEASE_NAME='ocp-release'
+export OCP_RELEASE='4.5.2-x86_64'
+export LOCAL_REGISTRY='my-disconnected-registry.example.com:5000'
+export LOCAL_SECRET_JSON='/path/to/pull_secret.json'
+
+oc adm -a ${LOCAL_SECRET_JSON} release mirror \
+--from=${UPSTREAM_REGISTRY}/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE} \
+--to=${LOCAL_REGISTRY}/ocp4 \
+--to-release-image=${LOCAL_REGISTRY}/ocp4/release:${OCP_RELEASE}
+
+export OCP_RELEASE='4.5.3-x86_64'
+
+oc adm -a ${LOCAL_SECRET_JSON} release mirror \
+--from=${UPSTREAM_REGISTRY}/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE} \
+--to=${LOCAL_REGISTRY}/ocp4 \
+--to-release-image=${LOCAL_REGISTRY}/ocp4/release:${OCP_RELEASE}
+```
+
+
+## Deploy UpdateService instance
+
+```
+cat create-updateservice-instance.yaml
+---
+apiVersion: updateservice.operator.openshift.io/v1
+kind: UpdateService
+metadata:
+  name: karampok
+  namespace: openshift-update-service
+spec:
+  replicas: 1
+  releases: "{DISCONNECTED_REGISTRY}/ocp4/release"
+  #releases: "{DISCONNECTED_REGISTRY}/ocp4 // OCP Release Image and Release Content in the same path
```

Comment:
Lets remove this, we should never recommend mixing release images and content in one repo, as it would most certainly make image scraping complicated